unfoldingWord/translationCore
# Windows Installation Improvements

#### This pull request addresses:

This adds a nicer splash image when installing tC.

#### How to test this pull request:

Make sure all the tests pass, then try out a windows build.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2886)
